submittedResponses.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_dynamodb_table" "submitted-responses-table" {
+  name           = "${var.env}-SubmittedResponses"
+  read_capacity  = 20
+  write_capacity = 20
+  hash_key       = "tx_id"
+
+  attribute {
+    name = "tx_id"
+    type = "S"
+  }
+
+  ttl {
+    attribute_name = "valid_until"
+    enabled = true
+  }
+}
```

Comment:
You should also add Name and Environment tags for this table too. e.g. 
```
tags {
    Name        = "${var.env}-submitted-responses"
    Environment = "${var.env}"
  }
```

This is just a best practice that you can use for cost allocation etc